bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/PlatformAPIImpl.java

```diff
@@ -655,9 +644,7 @@ private void createDefaultTenant(final PlatformServiceAccessor platformAccessor,
             sessionAccessor.deleteSessionId();
             sessionAccessor.setSessionInfo(session.getId(), tenantId);// necessary to create default data source
 
-            // Create default profiles
-            createDefaultProfiles(tenantServiceAccessor);
-
+            // Create default profiles: done by profile updater restart handler
```

Comment:
Since in SP default profiles are created in tenant creation we should call the creation here also in order to be consistent